### PR TITLE
WIP: Update implementation of EasyConfig to use ChainMap

### PR DESCRIPTION
**Pull request by cthoyt**

### CODE REVIEW

File: src/easy_config/__init__.py [hunk 1]
```diff
@@ -189,21 +190,13 @@ def load(
...
-            file_name = os.environ.get(envvar)
-            if file_name:
-                values.update(cls._load_file(file_name))
-        if _parse_environment:
-            values.update(cls._load_environment())
-        values.update(cls._load_dict(kwargs))
+        values = ChainMap(*cls._load_helper(
+            _additional_files=_additional_files,
+            _parse_files=_parse_files,
+            _parse_environment=_parse_environment,
```
Comment (scolby33) [inline src/easy_config/__init__.py:196]: @cthoyt it was broken because you missed passing this.

Comment (cthoyt) [reply src/easy_config/__init__.py:196]: LOL

**Comment by cthoyt** [comment]
merge this?

File: src/easy_config/__init__.py (hunk 1 above)
Comment (scolby33) [reply src/easy_config/__init__.py:196]: Right? Of course it was that.

**Pull request merged**